.github/workflows/build.yml

```diff
@@ -0,0 +1,71 @@
+name: Java CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
```

Comment:
Rather than validating the gradle wrapper seperately in each job, would it be better to have a seperate job that solely validates the gradle wrapper binary? That separate job could be made required.

Reply:
Yes, that sounds good :+1: Do you know out of your head how to do that, or can we do that as follow up?

Reply:
I don't know if this would immediately work so it can be left as a follow-up.

Reply:
why does it make sense to validate the gradle wrapper all the time? it should be sufficient to do it where the release terasoloty binary is built, isn't it?